src/assets/wise5/components/table/edit-table-advanced/edit-table-advanced.component.html

```diff
@@ -111,6 +111,32 @@
           {{ ::'table.canStudentEditAxisLabels' | translate }}
         </md-checkbox>
       </div>
+      <div>
+        <p>Fix Data to Table Column <span><md-icon>help</md-icon><md-tooltip style="height: auto" md-direction="right">You can author x or y data to always use a specific table column.<br/>The student will not be able to change the table column for that<br/>x or y data. If you want the student to be able to choose the table<br/>column, you can leave the select option blank for that x or y data.</md-tooltip></span></p>
```

Comment:
The text here (and in some other places) isn't translatable. Maybe we could wait until this is upgraded to Angular to add `$localize` tags?